CHANGELOG.md

```diff
@@ -2,11 +2,15 @@
 
 For advice on how to use these release notes see [our guidance on staying up to date with changes](https://frontend.design-system.service.gov.uk/staying-up-to-date/).
 
-## Unreleased
+## 5.2.0 (feature release)
 
-### New features
+In this release, we’ve adjusted our responsive type scale, which is available behind a feature flag. The type scale change is to make text easier to read on smaller screens. We’ve also deprecated the `useTudorCrown` parameter.
+
+To install this version with npm, run `npm install govuk-frontend@5.2.0`. You can also find more information about [how to stay up to date](https://frontend.design-system.service.gov.uk/staying-up-to-date/#updating-to-the-latest-version) in our documentation.
+
+## New features
```

Comment:
The heading levels are wrong from this point onwards – everything needs to be dropped a level.